tests/browser/elements.js

```diff
@@ -180,3 +180,26 @@ test('allow objects to be passed', function (t) {
   t.ok(result.outerHTML.indexOf('<div foo="bar">hey</div>') !== -1, 'contains foo="bar"')
   t.end()
 })
+
+test('supports extended build-in elements', function (t) {
+  t.plan(1)
+
+  var originalCreateElement = document.createElement
+  var optionsArg
+
+  // this iife is a must to avoid illegal invocation type errors, caused by transformed nanohtml tests
+  (function () {
```

Comment:
If tape would have `before`/`after` or `setup`/`teardown` support it could be implemented cleaner